plugin/parser/ast/create_schema_stmt.go

```diff
@@ -0,0 +1,11 @@
+package ast
+
+// CreateSchemaStmt is the struct for create schema statement.
+type CreateSchemaStmt struct {
+	ddl
+
+	Name           string
+	IfNotExists    bool
+	RoleSpec       *RoleSpec
+	SchemaElements []Node
```

Comment:
```suggestion
	SchemaElementList []Node
```
In the Bytebase, we should use the `List` instead of the plural representation.